scripts/collect_icon_testdata.sh

```diff
@@ -41,7 +41,7 @@ python $PROBTEST_DIR/probtest.py init \
     --file-id NetCDF "*atm_3d_hl*.nc" \
     --file-id NetCDF "*atm_3d_pl*.nc" \
     --reference probtest_output \
-    --member_ids 1,2 || exit 1
+    --member-num 2 || exit 1
```

Comment:
Is the feature of selected members tested anywhere? Perhaps it could be done the following way.
```suggestion
    --member-num 1,2 || exit 1
# Pass a list to member-mum to check ability to generate selected members
```
Or maybe even with non consecutive numbers.

Reply:
Yes, I think that would be good. I need to generate new test data for that and will update the hashes in a second.

Reply:
That was actually not as straight forward as I expected it to be but it should be solved now!

Reply:
Thank you. I've just approved this PR.